components/script/dom/gpudevice.rs

```diff
@@ -214,6 +220,139 @@ impl GPUDevice {
         }
     }
 
+    fn parse_render_pipeline<'c>(
+        &self,
+        descriptor: &GPURenderPipelineDescriptor,
+    ) -> (
+        Option<(PipelineLayoutId, Vec<BindGroupLayoutId>)>,
+        Option<RenderPipelineDescriptor<'c>>,
```

Comment:
`c is not related to any method argument. Is that intentional? What is the risk if the output lifetime is completely made up?

Reply:
Yes, it's intentional to make it compile. Output lifetime is automatically determined by compiler (to be 'static, but the compiler cannot determine that from function signature), so rewrote it to static lifetime to make it more obvious.